reco_utils/dataset/numpy_splitters.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Thanks for the comment. The splitter does not use TF, only the RBM algo. For the latter, the TF version is printed when initializing the model, i.e. `model = RBM( )`. Let me know if this answer your question.

Reply:
Sorry, I put the comment in the wrong place, for some reason I couldn't comment on the notebook. 
In the NCF notebook, Aaron added TF in the notebook, please take a look at this: https://github.com/Microsoft/Recommenders/blob/staging/notebooks/02_model/ncf_deep_dive.ipynb.

Reply:
I see what you mean. The point is that TF is not used explictly in the notebook as it is loaded only in the RBM class. I could load it in the notebook as well but this makes the first cell even slower to load. If you think printing the version here instead of at the moment the algo runs is better, I will change it accordingly.